### ISSUE TITLE
WebSocket Connect Info doesn't update if you hit [Show Connect info] before you hit [apply] leading to user confusion

### ISSUE THREAD

**Issue by BarryCarlyon** [obsproject/obs-studio#10947]
### Operating System Info

macOS 14 _and_ windows

### OBS Studio Version

30.2.0-rc1

### OBS Studio Log URL

N/A

### Expected Behavior

"Show Connect Info" in the Websocket Server Settings UI to be accurate

### Current Behavior

It's not accurate

### Steps to Reproduce

1. Open WebSocket Server Settings
2. Hit "Show Connect Info" observe the password
3. Close the Connect info prompt
4. Hit Generate password
5. Hit "Show Connect Info" observe the password
6. See that the password shown is the _old_ password (and the QR code not likely to be update)

### Anything else we should know?

Same issue _if_ you put a manual password in the Server Password UI and hit "show connect info" the old generated password is visible

The same bug is present in 30.1.2 both Windows/mac, UNLESS you hit apply.

Show Connect Info should Imply [apply] as otherwise users will hit generate password -> show connect info and forgot to apply and then have the wrong password on their clipboard. I only realised this when I cross checked windows as on windows ok/cancel/apply are all together but mac has apple on the far left

**TLDR:** [Show Connect Info] should imply an [Apply]

Additionally since related

The alignment of things on the info popup is "bad"

![image](https://github.com/obsproject/obs-studio/assets/20999/f2f599e2-d2cb-41d6-9f60-51045505abb3)

**Comment by BarryCarlyon** [comment]
Misfiled
